exposed-java-time/src/test/kotlin/org/jetbrains/exposed/DefaultsTest.kt

```diff
@@ -194,9 +194,11 @@ class DefaultsTest : DatabaseTestsBase() {
 
     @Test
     fun testDefaults01() {
+        println("Start test")
         val currentDT = CurrentDateTime
         val nowExpression = object : Expression<LocalDateTime>() {
             override fun toQueryBuilder(queryBuilder: QueryBuilder) = queryBuilder {
+                println("currentDialectTest = $currentDialectTest")
```

Comment:
Checking whether these `println()` were accidentally left in.

Reply:
thanks, will drop